manager/config/integration-tests/deployment_configmap.yaml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2020 IBM Corp.
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: m4d-config
+data:
+  SECRET_PROVIDER_URL: "http://secret-provider.m4d-system.svc.cluster.local:5555/get-secret"
+  SECRET_PROVIDER_ROLE: "demo"
+  #catalog connector
+  CATALOG_CONNECTOR_URL: "data-catalog:50085"
+  CATALOG_PROVIDER_NAME: "EGERIA"
+
+  #credentials manager connector
+  CREDENTIALS_CONNECTOR_URL: "data-catalog:50085"
+  CREDENTIALS_PROVIDER_NAME: "VAULT"
+
+  #policy-compiler and policy manager connectors
+  MAIN_POLICY_MANAGER_CONNECTOR_URL: "policycompiler:50090"
+  MAIN_POLICY_MANAGER_NAME: "MOCK"
+  USE_EXTENSIONPOLICY_MANAGER: "false"
+  #EXTENSIONS_POLICY_MANAGER_CONNECTOR_URL: "opa-connector:50082"
+  #EXTENSIONS_POLICY_MANAGER_NAME: "OPA"
+
+  #internal vault related variables
+  VAULT_ADDRESS: "http://vault.m4d-system.svc.cluster.local:8200/"
+  VAULT_DATASET_HOME: "m4d/dataset-creds/"
+  VAULT_USER_HOME: "m4d/user-creds/"
+  VAULT_DATASET_MOUNT: "v1/sys/mounts/m4d/dataset-creds"
+  VAULT_USER_MOUNT: "v1/sys/mounts/m4d/user-creds"
+  VAULT_TTL: "24h"
+  VAULT_AUTH: "kubernetes"
+  RUN_WITHOUT_VAULT: "1"
```

Comment:
RUN_WITHOUT_VAULT is not needed if the vault is installed as well. Otherwise, the credentials won't be stored, and the copy will fail.